### ISSUE TITLE
Lint script missing

### ISSUE THREAD

**Issue by anegg0** [wighawag/template-ethereum-contracts#8]
Hi @wighawag, and thank you for what you do, I can't tell you how much I use your work.

`yarn lint` returns `error Command "lint" not found`.
I couldn't find the script in your package.json either.
I'm unsure if I'm using the repo correctly.

**Comment by wighawag** [comment]
Oh I forgot to update the readme, I removed linting, I realised I am not a fan of it, while I like prettier, lint was getting in my way too much. Feel happier without it

**Comment by anegg0** [comment]
Not a problem, thank you for the update :)
I guess that closes this issue, won't do it myself since it's your repo!

**Comment by wighawag** [comment]
Yes true :D but happy to get feedback if you think lint is a good thing to have. could easily have a branch for example if needed

**Comment by anegg0** [comment]
I take care of the linting part on Emacs, so it's not really required on my end. 
The only things I struggled with would probably fit in other issues:
- package incompatibility with hardhat-toolbox (which I solved using yarn).
- launching with `npx hardhat` which wipes settings out (now I always make sure not to use it).

**Comment by wighawag** [comment]
closing as we revamp to use hardhat-deploy v2 alpha
